mne_bids/utils.py

```diff
@@ -365,29 +362,25 @@ def _estimate_line_freq(raw, verbose=False):
     tmin = 0
     tmax = int(min(len(raw.times), 10 * sfreq))
 
-    # get just five channels of data to estimate on
+    # get the data
     data = raw.get_data(start=tmin, stop=tmax,
-                        picks=picks, return_times=False)[0:5, :]
+                        picks=picks, return_times=False)
 
     # run a multi-taper FFT between Power Line Frequencies of interest
     psds, freqs = psd_array_welch(data, fmin=49, fmax=61,
                                   sfreq=sfreq, average="mean")
     usa_ind = np.where(freqs == min(freqs, key=lambda x: abs(x - 60)))[0]
     eu_ind = np.where(freqs == min(freqs, key=lambda x: abs(x - 50)))[0]
 
-    # get the average power within those frequency bands
+    # get the average and standard deviation power within those frequency bands
     usa_psd = np.mean((psds[..., usa_ind]))
     eu_psd = np.mean((psds[..., eu_ind]))
 
     if verbose is True:
         print("EU (i.e. 50 Hz) PSD is {} and "
               "USA (i.e. 60 Hz) PSD is {}".format(eu_psd, usa_psd))
 
-    if usa_psd > eu_psd:
-        line_freq = 60
-    else:
-        line_freq = 50
-    return line_freq
+    return 60 if usa_psd > eu_psd else 50
```

Comment:
@alexrockhill I will proceed as is but all the changes is this file were here unnecessary.